external-crates/move/crates/move-vm-config/src/runtime.rs

```diff
@@ -3,6 +3,14 @@
 
 use crate::verifier::{VerifierConfig, DEFAULT_MAX_CONSTANT_VECTOR_LEN};
 use move_binary_format::file_format_common::VERSION_MAX;
+use once_cell::sync::Lazy;
+
+#[cfg(feature = "gas-profiler")]
+const MOVE_VM_PROFILER_ENV_VAR_NAME: &str = "MOVE_VM_PROFILE";
+
+#[cfg(feature = "gas-profiler")]
+static PROFILER_ENABLED: Lazy<bool> =
+    Lazy::new(|| std::env::var(MOVE_VM_PROFILER_ENV_VAR_NAME).is_ok());
```

Comment:
Is this still used? Can't seem to see where this/`PROFILER_ENABLED` are used anywhere still.

Reply:
It can be used by someone who wants to profile a test

Reply:
Ah great point! I hadn't considered that case, but definitely makes sense to keep it for that!